.github/workflows/conformance.yml

```diff
@@ -0,0 +1,111 @@
+name: conformance
+
+on:
+  push:
+     branches:
+      - "*-conformance"
+      - "conformance-*"
```

Comment:
I considered this, but it felt odd since I was thinking about working on x feature/bugfix but _also wanting to run conformance tests_ vs. a branch specifically for working on the conformance tests themselves. The `release/*` branches have a very specific purpose where I'm thinking of conformance as some extra CI that I'd like to run on my branch.

I guess in either case you're probably going to push up a second branch to run conformance tests for your change? Not sure. Curious for your thoughts.

Reply:
> working on x feature/bugfix but also wanting to run conformance tests

I think if this is a common enough case, we could just run them on all PRs, or use something like a GitHub label (could probably grab the inverse of the `no-changelog` logic from https://github.com/hashicorp/consul-api-gateway/pull/101) that would be easier to add if it was determined to be relevant instead of renaming the branch for an open PR.

Reply:
Adjusted to run on `conformance/*` branches and on PRs with the label `pr/run-conformance` 🚀